extensions/cornerstone-dicom-sr/src/utils/addSRAnnotation.ts

```diff
@@ -0,0 +1,72 @@
+import { Types, annotation } from '@cornerstonejs/tools';
+import { metaData } from '@cornerstonejs/core';
+
+import getRenderableData from './getRenderableData';
+import toolNames from '../tools/toolNames';
+
+export default function addSRAnnotation(measurement, imageId, frameNumber) {
+  let toolName = toolNames.DICOMSRDisplay;
+
+  const measurementData = {
+    TrackingUniqueIdentifier: measurement.TrackingUniqueIdentifier,
+    renderableData: {},
+    labels: measurement.labels,
+    imageId,
+  };
+
+  measurement.coords.forEach(coord => {
+    const { GraphicType, GraphicData, ValueType } = coord;
+
+    if (measurementData.renderableData[GraphicType] === undefined) {
+      measurementData.renderableData[GraphicType] = [];
+    }
+
+    measurementData.renderableData[GraphicType].push(
+      getRenderableData(GraphicType, GraphicData, ValueType, imageId)
+    );
+  });
+
+  const { ValueType, GraphicType } = measurement.coords[0];
+
+  /** TODO: Add support for other SCOORD3D graphic types */
```

Comment:
Add to the comment something like: 
Read the tool name from the DICOM SR identification type in the future.